### ISSUE TITLE
[Feature request] Add id field in MtxGridColumn

### ISSUE THREAD

**Issue by sanioooook** [ng-matero/extensions#149]
Can you add `id` field in `MtxGridColumn`?
Like `id: (rowData, index) => string` or simple `id: string`

**Comment by nzbin** [comment]
Can you tell me why you need an `id` field?

**Comment by sanioooook** [comment]
e2e tests in project are oriented to ids

**Comment by nzbin** [comment]
14.5.0 has added id field, please have a try.

**Comment by sanioooook** [comment]
it seems to me that I have not fully expressed my thought.
can you also add the use of this id in the cells? Like [id]="col.id"

**Comment by sanioooook** [comment]
In tests, I need to get an element using a selector(`$('css-selector')`), take the text from the element and compare it with what I expect

**Comment by nzbin** [comment]
> it seems to me that I have not fully expressed my thought. can you also add the use of this id in the cells? Like `[id]="col.id"` for all `<span>` in [cell.component.html](https://github.com/ng-matero/extensions/blob/master/projects/extensions/grid/cell.component.html)

The id attribute is unique, it shouldn't be added to multiple elements.

**Comment by nzbin** [comment]
> In tests, I need to get an element using a selector(`$('css-selector')`), take the text from the element and compare it with what I expect

You can use custom column's class to get the element.

**Comment by sanioooook** [comment]
> > it seems to me that I have not fully expressed my thought. can you also add the use of this id in the cells? Like `[id]="col.id"` for all `<span>` in [cell.component.html](https://github.com/ng-matero/extensions/blob/master/projects/extensions/grid/cell.component.html)
> 
> The id attribute is unique, it shouldn't be added to multiple elements.

yes, I know, that's why I suggested `id: (rowData, index) => string` so that the id would be unique
